devops/arangodb.yml

```diff
@@ -9,6 +10,10 @@
     - name: arangodb_root_password
       prompt: "Password for ArangoDB root account"
 
+    - name: ssl_cert_path
+      prompt: "Path to parent ssl cert directory"
+      default: "/etc/letsencrypt/live/multinet.app/"
```

Comment:
Why is this path the default?

Reply:
Just for speed when we're reprovisioning. Since this filepath will always be the same for us, I figured we could save a little time

Reply:
It also helps us with the vagrant set up, since vagrant doesn't prompt for these. It will take the default as the argument

Reply:
Ok, makes sense if that's where the LE utility places the file when run locally.